### PR TITLE
Make protocol implicit

### PR DESCRIPTION
**Pull request by pmikolajczyk41**
After some suggestions, we make passing protocol (either `ws` or `wss`) implicit. This means that if you want to connect without ssl, you can either pass a bare address like `127.0.0.1:9943` or `ws://127.0.0.1:9943`. In order to use ssl, pass `wss://127.0.0.1:9943`.

### CODE REVIEW

File: aleph-client/src/lib.rs [hunk 1]
```diff
@@ -51,39 +51,23 @@ pub type Header = GenericHeader<BlockNumber, BlakeTwo256>;
...
-pub fn from(use_ssl: bool) -> Protocol {
-    match use_ssl {
-        true => Protocol::WSS,
-        false => Protocol::WS,
-    }
-}
-
-pub fn create_connection(address: &str, protocol: Protocol) -> Connection {
-    create_custom_connection(address, protocol).expect("connection should be created")
+    format!("ws://{}", address)
```
Comment (fbielejec) [inline aleph-client/src/lib.rs:63]: what about `impl Display for Protocol` and `Protocol::WS.to_string();`

Comment (pmikolajczyk41) [reply aleph-client/src/lib.rs:63]: You mean reviving `Protocol` enum and use its string representation instead of in-place strings?

Comment (fbielejec) [reply aleph-client/src/lib.rs:63]: yes

Comment (pmikolajczyk41) [reply aleph-client/src/lib.rs]: done

**Review by Marcin-Radecki — APPROVED** [review]
Very nice :)

**Review by fbielejec — APPROVED** [review]

**Pull request merged**